weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/ValueViewNumberTimestamp.tsx

```diff
@@ -7,18 +7,32 @@ const JAN_1_2100_S = 4_102_444_800;
 const JAN_1_2000_MS = 1000 * JAN_1_2000_S;
 const JAN_1_2100_MS = 1000 * JAN_1_2100_S;
 
-// TODO: This is only looking at value, but we could also consider the value name, e.g. "created".
+// Regex pattern to match likely timestamp field names
+const TIMESTAMP_PATTERN =
```

Comment:
I wish JS had x-mode - in its absence perhaps you could either build this from parts or add a comment explaining the pieces for better readability.